src/x11/extensions/xkb.cpp

```diff
@@ -64,6 +64,16 @@ const string keyboard::layout_name(size_t index) const {
   return layouts[index].symbols[index];
 }
 
+/**
+ * Get current variant name
+ */
+const string keyboard::variant_name(size_t index) const {
+  string group_name = this->group_name(index);
+  int start = group_name.find('(') + 1;
+  int num_chars = group_name.find(')') - start;
+  return group_name.substr(start, num_chars);
```

Comment:
could you add the string that you expect to find as `group_name` to the comment above? I guess that a real unit-test is hard, but a comment about the expectations would be helpful to understand this method.